examples/03-tips-n-tricks/01-smoothing-element-size-transitions.py

```diff
@@ -0,0 +1,108 @@
+"""
+.. _ref_element_size_transition:
+
+MAPDL element size transition example
+--------------------------------------------
+
+Often you may wish to view an area, or region, in
+increased detail using a smaller element size. This
+usually introduces a sharp element size transition
+in the mesh. This example demonstrates one way to
+reduce this effect using a simple geometry.
+
+First, start MAPDL as a service.
+"""
+# sphinx_gallery_thumbnail_number = 3
+
+from ansys.mapdl.core import launch_mapdl
+
+mapdl = launch_mapdl()
+mapdl
+
+###############################################################################
+# The Geometry (a box)
+# ~~~~~~~~~~~~~~~~~~~~
+# This example uses very simple geometry. In this case, a 3D box with 4 sides.
+# The two remaining sides are open, and we have a rectangular tube. Its
+# dimensions are 5 x 5 x 1. We create this using 8 keypoints and then
+# building 4 areas.
+#
+
+mapdl.prep7()
+
+k0 = mapdl.k(1, 0, 0, 0)
+k1 = mapdl.k(2, 0, 5, 0)
+k2 = mapdl.k(3, 5, 5, 0)
+k3 = mapdl.k(4, 5, 0, 0)
+
+k4 = mapdl.k(5, 0, 0, 1)
+k5 = mapdl.k(6, 0, 5, 1)
+k6 = mapdl.k(7, 5, 5, 1)
+k7 = mapdl.k(8, 5, 0, 1)
+
+a0 = mapdl.a(1, 2, 3, 4)
+a1 = mapdl.a(5, 6, 7, 8)
+a2 = mapdl.a(3, 4, 8, 7)
+a3 = mapdl.a(1, 2, 6, 5)
+
+
+###############################################################################
+# Mesh size
+# ~~~~~~~~~
+# We set the global mesh size to 0.7, but we're particularly interested in
+# face a2, so we set the element size of that face to 0.1. Then we just
+# need to specify an element type (we use SHELL181 here) and mesh our shape!
+#
+# Additionally, we specify that we want triangular elements using mshape.
+# This is purely to demonstrate the effect! This will work just as well
```

Comment:
```suggestion
# Additionally, we specify that we want triangular elements using ``mshape``.
# This is purely to demonstrate the effect. This will work just as well
```